frontend/http/frontend.go

```diff
@@ -163,7 +164,16 @@ func (t *Frontend) scrapeRoute(w http.ResponseWriter, r *http.Request, _ httprou
 		return
 	}
 
-	resp, err := t.logic.HandleScrape(context.Background(), req)
+	host, _, err := net.SplitHostPort(r.RemoteAddr)
+	if err != nil {
+		WriteError(w,err)
```

Comment:
not formatted -.-

`WriteError` checks if it is a `bittorrent.ClientError`, so I  didn't care about that. Will add a line of logging.